game/static/game/css/board.css

```
@@ -787,6 +787,50 @@ body {
+    border-radius: 10px;
+    font-size: 0.9rem;
+    margin-bottom: 12px;
+    display: flex;
+    align-items: center;
+    gap: 12px;
+    box-shadow: 0 8px 24px rgba(0, 0, 0, 0.4);
+    animation: slideIn 0.4s ease forwards, fadeOut 0.5s ease 4.5s forwards;
+    backdrop-filter: blur(8px);
+    text-align: left;
+}
+@keyframes slideIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)